spec/models/site_spec.rb

```diff
@@ -24,4 +24,22 @@
   describe 'associations' do
     it { should belong_to(:provider) }
   end
+
+  describe '#touch_provider' do
```

Comment:
"touch provider sites" might be more accurate?

Reply:
You are already in the context if a `site` so I think touching the provider is accurate

Reply:
Actually I think @timabell is right, since we don't actually call `touch_provider` here, we're testing the after `after_save` hook, and also by the `touch_provider` method coincidentally. But at this point we just need to get this merged so we can deal with this later, if at all.